build-system/compile/bento-remap.js

```diff
@@ -66,11 +66,18 @@ const getAllRemappings = once(() => {
   }));
 
   // Allow component cross-dependency
-  const componentRemappings = bentoBundles.map(({name, version}) => ({
-    source: `./src/bento/components/${name}/${version}/${name}`,
-    cdn: `./${name}-${version}.mjs`,
-    npm: `@bentoproject/${getNameWithoutComponentPrefix(name)}`,
-  }));
+  const componentRemappings = bentoBundles.map(({name, version}) => {
+    const nameWithoutPrefix = getNameWithoutComponentPrefix(name);
+    return {
+      source: `./src/bento/components/${name}/${version}/${name}`,
+      cdn: `./${name}-${version}.mjs`,
+      npm:
+        // Special: NPM builds depend on `mustache` directly
+        nameWithoutPrefix === 'mustache'
```

Comment:
nit: seems like it could be cleaner to be able to declare in the bundle config that extension `A` should actually be redirected to B (where B is another extension or another npm package etc). That way we don't have a special case for `mustache` in the bento-remap script

Reply:
(not blocking this PR)